### ISSUE TITLE
BratWriter: line ending may break output

### ISSUE THREAD

**Issue by pkluegl** [dkpro/dkpro-core#1089]
BratWriter line 608: If the covered Text of the annotation contains a line break, the resulting file is broken? The reason seems to be that the tokenizer is not robust to OS-specific line ending styles.

**Comment by reckart** [comment]
Which tokenizer? In brat or in DKPro Core?

**Comment by pkluegl** [comment]
I don't know. there is some problem (using Windows) in the BratReaderWrtierTest.testConll2009_2. The token  51 includes the linebreak, which is propagated to the written file. I assume line breaks are not supported there, right?

**Comment by reckart** [comment]
Seems to be related to this issue in brat: "Allow any character in textbound annotation span" https://github.com/nlplab/brat/issues/819

But in our case, it is strange that the line break crops up inside the annotation in the first place. CoNLL 2009 files also do not support line breaks. So maybe we have a Windows-specific bug already in the CoNLL reader?

**Comment by pkluegl** [comment]
Yes, maybe. I created this issue since I skipped this problem while working on #414, as it looked like it could be more than just a non-windows-compatible test.

**Comment by reckart** [comment]
AFAIK windows is the only remaining system with two-letter linebreak indicators. Linux and OS X both use `\n` these days.

**Comment by pkluegl** [comment]
It's not like I could do something about it ;-) 
Ok, I could switch my OS, but to be (build-) compatible with Windows would still be desirable in my opinion.

**Comment by reckart** [comment]
Lol :) Unfortunately, our Windows-based build is currently failing before it reaches that module.

**Comment by pkluegl** [comment]
Well, the reaons why I created this issue and not simple fixing the test: What would happen if the tokenizer creates a token containing a linebreak. Happened before. Would there be a problem in the writer creating invalid brat?

**Comment by pkluegl** [comment]
oh there are so many failures :-(

**Comment by reckart** [comment]
There is no tokenizer involved in `BratReaderWriterTest.testConll2009_2`. Actually there is not even a Conll2009 reader involved. If I remember correctly, then `testConll2009_2` basically takes the brat file produced in `testConll2009`, reads it, and writes it again as brat to check that a brat-round-trip works without a problem.

**Comment by reckart** [comment]
Getting a Windows-compatible build is surely the goal. After all, we finally set up a Windows-based build slave for that :)

**Comment by reckart** [comment]
Does this test still fail for you?

**Comment by pkluegl** [comment]
nope, test is ok now
